osu.Game/Overlays/Profile/Sections/Historical/UserHistoryGraph.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static osu.Game.Users.User;
+
+namespace osu.Game.Overlays.Profile.Sections.Historical
+{
+    public abstract class UserHistoryGraph : UserGraph<DateTime, long>
```

Comment:
I just noticed this while scanning for implementors of `UserGraph` - is there a good reason this is abstract?

Reply:
Need to check my follow-up pr: there was a reason, but at this point I'm not sure. If there's no reason - will remove.

Reply:
Tooltip is the only reason. We have two `UserHistoryGraph`s in the overlay with different tooltips
![Screenshot_1](https://user-images.githubusercontent.com/22874522/76258046-e31f8700-6263-11ea-9b62-60c04833cc98.png)
![Screenshot_2](https://user-images.githubusercontent.com/22874522/76258047-e3b81d80-6263-11ea-97d5-f1b24f6999f2.png)

Reply:
Overriding an entire class to change some text seems like overkill. They're semantically a little different but can be always split if necessary. I'd advocate for publicly settable property instead of having an overridable property but at this point there's so many conventions I'm not going to insist on it.